cmd/root.go

```diff
@@ -63,6 +65,16 @@ func NewRootCmd() *cobra.Command {
 
 Welcome to the Astro CLI, the modern command line interface for data orchestration. You can use it for Astro, Astronomer Software, or Local Development.`,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			// Check for latest version
+			if config.CFG.UpgradeMessage.GetBool() {
+				// create github client
+				githubClient := github.NewClient(nil)
+				// compare current version to latest
+				err = version.CompareVersions(githubClient, "astronomer", "astro-cli")
+				if err != nil {
+					logrus.Debug(err)
```

Comment:
you would need to add this log to the `InitDebugLogs`, directly writing logrus.Debug won't work.
```suggestion
					softwareCmd.InitDebugLogs = append(softwareCmd.InitDebugLogs, fmt.Errorf("Error comparing CLI versions: %s", err.Error()))
```